### PR TITLE
add javascript.js and main.css

### PR DESCRIPTION
**Pull request by mortaen**
- add javascript.js and main.css to display content on one page
- move all html to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,113 +1,226 @@
...
+          name="answer"
+          rows="7"
+          cols="35"
+          placeholder="Enter your answer here..."
+        ></textarea>
+
+        <label class="form__label" for="tags">Tags</label>
+        <textarea
+          class="form__textarea"
+          id="question"
```
Comment (marcbendixen) [inline index.html:166]: IDs need to be unique. There are two `id="question"`

File: javascript.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  profileMain.classList.add('main--hidden')
+  createMain.classList.remove('main--hidden')
+})
+
+profileButton.addEventListener('click', () => {
+  homeMain.classList.add('main--hidden')
+  bookmarksMain.classList.add('main--hidden')
+  createMain.classList.add('main--hidden')
+  profileMain.classList.remove('main--hidden')
+})
```
Comment (marcbendixen) [inline javascript.js:37]: A better naming convention for this file would be `main.js` or `script.js` instead of `javascript.js`.

**Review by marcbendixen — COMMENTED** [review]
Good job 🥳 
If you don't need the `create.html` anymore, you should delete it to keep the project clean.

File: css/components/footer.css [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 100%;
 }
 
 .navbar__link {
-    text-decoration: none;
-    
+  text-decoration: none;
```
[the remark this answers is not in the document]
Comment (mortaen) [reply css/components/footer.css:14]: Thanks, I've removed it.

File: css/components/footer.css [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
-    font-size: 48px; 
+.material-icons.md-48 {
+  font-size: 48px;
 }
 
 .material-icons.md-light {
-     color: rgba(255, 255, 255, 1); 
-    }
\ No newline at end of file
+  color: rgba(255, 255, 255, 1);
```
[the remark this answers is not in the document]
Comment (mortaen) [reply css/components/footer.css:29]: Thanks, changed it.

File: index.html (hunk 1 above)
Comment (mortaen) [reply index.html:217]: Thanks, removed them. They weren't used anyway 😄

File: javascript.js (hunk 2 above)
Comment (mortaen) [reply javascript.js]: Thanks, it is now called main.js.

File: css/components/main.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.main.main--hidden {
```
[the remark this answers is not in the document]
Comment (mortaen) [reply css/components/main.css:1]: Thanks, changed it.

**Pull request closed without merging**